{{cookiecutter.project_slug}}/.github/workflows/pythonapp.yml

```diff
@@ -0,0 +1,65 @@
+# This workflow will install Python dependencies, run tests and lint with multiple version of Python
+name: matchms
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
+    - name: Test with nlesc-template tests
+      run: |
+        python setup.py test
+    - name: Lint with flake8
```

Comment:
The template ships with [prospector](https://github.com/NLeSC/python-template/blob/3a00fb32f6adb607bce43493f107bb49e05e44e3/%7B%7Bcookiecutter.project_slug%7D%7D/project_setup.rst#coding-style-conventions-and-code-quality). It should be used in CI aswell instead of flake8.

Reply:
Seems fixed